sync-data-to-another-database/README.md

```diff
@@ -0,0 +1,195 @@
+# Synchronize Kuzzle with another database
+
+## Introduction
+
+Kuzzle uses Elasticsearch as a database, which allows it to offer very good search performance on large volumes.  
```

Comment:
As [Elasticsearch is not a database](https://www.elastic.co/blog/found-elasticsearch-as-nosql) but a search engine, I think this is not a good idea to explain that we use it as a database (officially, we do not).

Morevover, "more traditional databases" makes me think of RDBMS and I don't think that Cassandra fits that bill. But we can easily synchronize with RDBMS too. So it may be a good idea to rewrite that part to explain that we can synchronize with _any database system, whatever its architecture (RDBMS, column-oriented, NoSQL, graph, ...)_
This how-to demonstrates this using Cassandra, but we could as easily synchronize with MongoDB, OrientDB, MySQL, ...

Our official position is that Kuzzle handles hot data with maximum performances, and that it is easy to synchronize Kuzzle with databases, as we aim to demonstrate with that how-to.
There are many reasons to synchronize data with other databases, for instance if that data is critical, or if Kuzzle is to be integrated in a larger, already existing information system.

Reply:
@scottinet  I get your point about search engine vs database. But to me it is still a database, it stores data, albeit for search. To me both are valid, but it's true that if we say "Search Engine" it is easier to understand why we want to sync with another db.

How about:
Kuzzle uses Elasticsearch as a search engine...

and then
However, for robustness, you commonly want to use Elasticsearch with another database to store master records...

Reply:
@appacea > ES does store data, but databases have very strict requirements about data integrity and fault tolerance, and Elasticsearch does not yet comply to these requirements. It wouldn't be accurate (and misleading) to tell our users that ES is a database, and this isn't our philosophy anyway.

That said, your proposal seems compatible with my remarks. :+1: